common/src/main/java/io/netty/util/internal/EmptyArrays.java

```diff
@@ -22,23 +22,16 @@
 
 public final class EmptyArrays {
 
-    public static final byte[] EMPTY_BYTES = new byte[0];
-    public static final char[] EMPTY_CHARS = new char[0];
-    public static final boolean[] EMPTY_BOOLEANS = new boolean[0];
-    public static final double[] EMPTY_DOUBLES = new double[0];
-    public static final float[] EMPTY_FLOATS = new float[0];
-    public static final int[] EMPTY_INTS = new int[0];
-    public static final short[] EMPTY_SHORTS = new short[0];
-    public static final long[] EMPTY_LONGS = new long[0];
-    public static final Object[] EMPTY_OBJECTS = new Object[0];
-    public static final Class<?>[] EMPTY_CLASSES = new Class[0];
-    public static final String[] EMPTY_STRINGS = new String[0];
-    public static final StackTraceElement[] EMPTY_STACK_TRACE = new StackTraceElement[0];
-    public static final ByteBuffer[] EMPTY_BYTE_BUFFERS = new ByteBuffer[0];
-    public static final Certificate[] EMPTY_CERTIFICATES = new Certificate[0];
-    public static final X509Certificate[] EMPTY_X509_CERTIFICATES = new X509Certificate[0];
-    public static final javax.security.cert.X509Certificate[] EMPTY_JAVAX_X509_CERTIFICATES =
-            new javax.security.cert.X509Certificate[0];
+    public static final byte[] EMPTY_BYTES = {};
+    public static final char[] EMPTY_CHARS = {};
+    public static final Object[] EMPTY_OBJECTS = {};
+    public static final Class<?>[] EMPTY_CLASSES = {};
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Make this member "protected". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS2386)
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Make this array "private". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1873)